src/helpers/crypto.ts

```diff
@@ -1,15 +1,77 @@
 import * as nacl from "tweetnacl";
 import { HexString } from "@frequency-chain/ethereum-utils";
 import { u8aToHex } from "@polkadot/util";
+import {
+  SiwfResponseCredentialGraph,
+  SiwfResponseCredentialRecoverySecret,
+  VerifiedRecoverySecret,
+} from "@projectlibertylabs/siwf";
+import { multibaseSr25519 } from "./utils";
 
-export function generateGraphKeyPair(): {
-  privateKey: HexString;
-  publicKey: HexString;
+export function generateGraphKeyCredential(
+  accountId: string,
+  keyPair: nacl.BoxKeyPair,
+): SiwfResponseCredentialGraph {
+  const issuer = multibaseSr25519(accountId);
+  return {
+    "@context": [
+      "https://www.w3.org/ns/credentials/v2",
+      "https://www.w3.org/ns/credentials/undefined-terms/v2",
+    ],
+    type: ["VerifiedGraphKeyCredential", "VerifiableCredential"],
+    issuer: `did:key:${issuer}`,
```

Comment:
I think this is the change we need to make assuming that the `accountId` is the Ethereum address (which I think it is)
```suggestion
    issuer: `did:ethr:${accountId}`,
```